dts/bindings/adc/lltc,ltc2451.yaml

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2023 Brill Power Ltd. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+description: Linear Technology LTC2451 ADC
+
+compatible: "lltc,ltc2451"
+
+include: i2c-device.yaml
+
+properties:
+
```

Comment:
Useless line, otherwise LGTM.

Reply:
Has been removed, thanks for the review!